docs/reference/cli/filters.md

```diff
@@ -140,7 +140,7 @@ Active:
 > StatefulSet
 > PrometheusConfigReport
 > ReplicaSet
-Unused: 
+Unused:
```

Comment:
You make this update also in another PR https://github.com/k8sgpt-ai/docs/pull/95
Let's merge this one first, then the other one. Thank you